pkg/image/registry/imagestreamtag/rest.go

```diff
@@ -107,8 +118,216 @@ func (r *REST) Get(ctx kapi.Context, id string) (runtime.Object, error) {
 	return newISTag(tag, imageStream, image, false)
 }
 
+func (r *REST) Create(ctx kapi.Context, obj runtime.Object) (runtime.Object, error) {
+	istag, ok := obj.(*imageapi.ImageStreamTag)
+	if !ok {
+		return nil, kapierrors.NewBadRequest(fmt.Sprintf("obj is not an ImageStreamTag: %#v", obj))
+	}
+	if err := rest.BeforeCreate(Strategy, ctx, obj); err != nil {
+		return nil, err
+	}
+	namespace, ok := kapi.NamespaceFrom(ctx)
+	if !ok {
+		return nil, kapierrors.NewBadRequest("a namespace must be specified to import images")
+	}
+
+	// lifting source and hint information from annotations.  These will be dropped on a future get, so that's a thing
+	tagSource := istag.Annotations["source"]
```

Comment:
@openshift/api-review I think we really need a way to create an imagestreamtag directly, but specification of its source ends up non-obvious (at least to me).  I plumbed through the intent using annotations that aren't present when you do the create, but having a structure like this makes it easier for a client to express intent and not have to have all this logic client-side.

Do we want to do something like this or would we create yet another virtual object for the sole purpose of creating an istag?